### PR TITLE
consul-readability - seperate resources

### PR DESCRIPTION
**Pull request by jharshman**
make easier to read and edit by putting each resource in it's own file

### CODE REVIEW

**Comment by jharshman** [comment]
cla done

**Comment by jharshman** [comment]
hmm cla didn't go through.  refreshing.

File: stable/consul/templates/consul-service.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
```
Comment (mattfarina) [inline stable/consul/templates/consul-service.yaml:1]: The service used to be wrapped in `{{- if .Values.uiService.enabled }}`. Why was that removed?

**Review by mattfarina — DISMISSED** [review]
Thank you for breaking this up to make it more readable. I just have one outstanding bit that I'm guessing was an oversight.

**Comment by mattfarina** [comment]
@jharshman If you have done the CLA then commenting again will hopefully catch it.

File: stable/consul/templates/consul-service.yaml (hunk 1 above)
Comment (jharshman) [reply stable/consul/templates/consul-service.yaml:1]: It wasn't removed. This isn't the Consul UI Service.  That one is under `templates/ui-service.yaml` and is still wrapped in the conditional.

**Comment by jharshman** [comment]
update cla
screen shot 2017-12-27 at 12 40 35 pm

**Comment by jharshman** [comment]
@mattfarina Not really sure why the tests are failing.
